.storybook/themes/news.scss

```diff
@@ -340,6 +340,12 @@
 					font-size: var(--body-font-size-small),
 					line-height: var(--body-line-height-small),
 				),
+				divider: (
+					margin: var(--global-spacing-6),
```

Comment:
There should be no spacing values for the component - this would be driven by the consumer either using `grid` or `flex` to provide layout spacing between elements